somewherewarm/woocommerce-composite-products-conditional-images
# Added compatibility with the block-based single product template

closes #10 

### Testing steps

1. Activate Storefront. 
2. Create a Composite Product with a conditional images scenario. 
3. View the Composite and ensure that the main product image is updated when the scenario is matched. 
4. Switch the active theme to Tsubaki and ensure that the block-based single product template is used. 
5. View the Composite and ensure that the main product image is updated when the scenario is matched.
